src/libraries/InterestRateModule.sol

```diff
@@ -0,0 +1,60 @@
+/**
+ * Created by Arcadia Finance
+ * https://www.arcadia.finance
+ *
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity ^0.8.13;
+
+import {Owned} from "../../lib/solmate/src/auth/Owned.sol";
+import {DataTypes} from "./DataTypes.sol";
+
+abstract contract InterestRateModule is Owned {
+    uint256 public interestRate; //18 decimals precision
+
+    DataTypes.InterestRateConfiguration public interestRateConfig;
+
+    /**
+     * @notice Set's the configration parameters of InterestRateConfiguration struct
+     * @param newConfig New set of configration parameters
+     */
+
+    function setInterestConfig(DataTypes.InterestRateConfiguration calldata newConfig) external onlyOwner {
+        interestRateConfig = newConfig;
+    }
+
+    /**
+     * @notice Calculates the interest rate
+     * @param utilisation Utilisation rate
```

Comment:
Mention the decimal precision in all documentation

Reply:
@alikonuk1 can you pick this up?